ddpui/ddpdbt/dbthelpers.py

```diff
@@ -12,6 +12,46 @@
 
 logger = CustomLogger("ddphelpers")
 
+# Sensitive file patterns that should not be committed to git
+SENSITIVE_FILE_PATTERNS = [
```

Comment:
lets remove this 

- add a gitignore file while setting up a new project. we can all standard stuff but main dir to ignore `profiles/`, `profiles.yml`, `profiles.yaml`, `.env*` 
- while connecting to an exisitng git project with already .gitignore in place - we need verify if `profiles/` is there or not. if not update the .gitignore